### PR TITLE
Add warning prompt to private cloud rotate password commands and delete commands

### PR DESCRIPTION
**Pull request by jonathanhe-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vmware

**Review by wangzelin007 — APPROVED** [review]
LGTM

**Review by cataggar — APPROVED** [review]

**Comment by jonathanhe-msft** [comment]
@zhoxing-ms please review when you get the chance. There will be one additional PR in the future to upgrade to version 4.0.0.

File: src/vmware/azext_vmware/_params.py [hunk 1]
```diff
@@ -23,12 +23,14 @@ def load_arguments(self, _):
     with self.argument_context('vmware private-cloud') as c:
         c.argument('cluster_size', help='Number of hosts for the default management cluster. Minimum of 3 and maximum of 16.')
         c.argument('internet', help='Connectivity to internet. Specify "Enabled" or "Disabled".')
+        c.argument('yes', help='Deletes without confirmation.')
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_params.py:26]: Please use the first person voice
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```

File: src/vmware/azext_vmware/_params.py [hunk 2]
```diff
@@ -23,12 +23,14 @@ def load_arguments(self, _):
...
         c.argument('cluster_size', help='Number of hosts for the default management cluster. Minimum of 3 and maximum of 16.')
         c.argument('internet', help='Connectivity to internet. Specify "Enabled" or "Disabled".')
+        c.argument('yes', help='Deletes without confirmation.')
 
     with self.argument_context('vmware cluster') as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the cluster.')
         c.argument('sku', help='The product SKU.')
         c.argument('size', help='Number of hosts for the cluster. Minimum of 3 and a maximum of 16.')
         c.argument('hosts', nargs='+', help='A cluster\'s hosts in the private cloud.')
+        c.argument('yes', help='Deletes without confirmation.')
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_params.py:33]:
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```

File: src/vmware/azext_vmware/_params.py [hunk 3]
```diff
@@ -47,10 +49,10 @@ def load_arguments(self, _):
 
     with self.argument_context('vmware private-cloud delete') as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the private cloud.')
-        c.argument('yes', help='Deletes without confirmation.')
 
     with self.argument_context('vmware authorization') as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the authorization.')
+        c.argument('yes', help='Deletes without confirmation.')
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_params.py:55]:
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```

File: src/vmware/azext_vmware/_params.py [hunk 4]
```diff
@@ -105,11 +107,13 @@ def load_arguments(self, _):
 
     with self.argument_context('vmware hcx-enterprise-site') as c:
         c.argument('name', options_list=['--name', '-n'], help='The name of the HCX Enterprise Site.')
+        c.argument('yes', help='Deletes without confirmation.')
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_params.py:110]:
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```

File: src/vmware/azext_vmware/_params.py [hunk 5]
```diff
@@ -105,11 +107,13 @@ def load_arguments(self, _):
 
     with self.argument_context('vmware hcx-enterprise-site') as c:
         c.argument('name', options_list=['--name', '-n'], help='The name of the HCX Enterprise Site.')
+        c.argument('yes', help='Deletes without confirmation.')
 
     with self.argument_context('vmware datastore') as c:
         c.argument('name', options_list=['--name', '-n'], help='The name of the datastore.')
         c.argument('cluster', help='The name of the cluster.')
         c.argument('lun_name', help='Name of the LUN to be used.')
+        c.argument('yes', help='Deletes without confirmation.')
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_params.py:116]:
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```

File: src/vmware/azext_vmware/_params.py [hunk 6]
```diff
@@ -126,6 +130,7 @@ def load_arguments(self, _):
 
     with self.argument_context('vmware addon') as c:
         c.argument('name', options_list=['--name', '-n'], help='Name of the addon.')
+        c.argument('yes', help='Deletes without confirmation.')
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_params.py:133]:
```suggestion
        c.argument('yes', help='Delete without confirmation.')

```

**Comment by zhoxing-ms** [comment]
Do you need to modify the `setup.py` to release the new version `4.0.0`?

**Comment by jonathanhe-msft** [comment]
> Do you need to modify the `setup.py` to release the new version `4.0.0`?

I will create another PR when we are ready to release the new 4.0.0 version with the modifications to `setup.py`

**Review by cataggar — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**